src/sentry/api/endpoints/organization_details.py

```diff
@@ -343,6 +436,15 @@ def save(self, *args, **kwargs):
 class OrganizationDetailsEndpoint(OrganizationEndpoint):
     doc_section = DocSection.ORGANIZATIONS
 
+    def append_trusted_relays_info(self, organization, context):
+        """Adds the trusted realy information to the deserialized data"""
+        trusted_relays_raw = (
+            organization.get_option("sentry:trusted-relays", TRUSTED_RELAYS_DEFAULT) or []
```

Comment:
```suggestion
            organization.get_option("sentry:trusted-relays") or TRUSTED_RELAYS_DEFAULT
```

Reply:
changed to 

     trusted_relays_raw = obj.get_option("sentry:trusted-relays") or []

since this is what I need

Reply:
done